packages/framework/src/Model/Product/Search/Export/ProductSearchExportWithFilterRepository.php

```diff
@@ -15,8 +16,13 @@
 use Shopsys\FrameworkBundle\Model\Product\ProductVisibility;
 use Shopsys\FrameworkBundle\Model\Product\ProductVisibilityRepository;
 
-class ProductSearchExportWithFilterRepository extends ProductSearchExportRepository
+class ProductSearchExportWithFilterRepository
```

Comment:
in the method `createQueryBuilder` the parameter `$locale` is never used. I have a feeling it could be removed because it is in the method only to fulfill the parent signature. But it would also mean an upgrade instruction...

So the decision is up to you